mitodl/ol-infrastructure
# Replacing pomerium with traefik

# Description (What does it do?)

Replace nginx and pomerium with traefik and forward-auth so that we can use Keycloak authentication with Airbyte. Additionally, this should be able to handle dagster instance to internally communicate with Airbyte using basic auth.

# How can this be tested?

Need to try out a pulumi test first and then possibly deploy it to CI to test and verify. Sure there are a few minor tweaks that are required especially around dagster and basic auth.

# Checklist

- [ ] Pulumi run
- [ ] Create keycloak OIDC client
- [ ] Add secrets to vault
- [ ] Deploy Airbyte CI for testing
